### ISSUE TITLE
Repl demo does not work anymore...

### ISSUE THREAD

**Issue by ZzubbuzZ** [yourlabs/django-autocomplete-light#1127]
Hi,
I just gave it a try this morning, and get this error :
```log
ImportError: cannot import name 'six' from 'django.utils' (/home/runner/.local/share/virtualenvs/python3/lib/python3.7/site-packages/django/utils/__init__.py)
```
BR

**Comment by Walek001** [comment]
Same issue is live demo hosted on repl site: https://dal--jpic.repl.co
<img width="828" alt="image" src="https://user-images.githubusercontent.com/32902745/70802760-9b977500-1db2-11ea-9543-08fd25203e67.png">

**Comment by jpic** [comment]
Yep I know, I'm working on my own repl-ish replacement (actually it's a replacement for Ansible/Docker/Compose that also provides a server you can register your git against).

**Comment by morenoh149** [comment]
Here to report the same. Guess its a bump.

**Comment by jpic** [comment]
Fixed in 824a20f985a6761a94692e820bbac9bb6e9e3029
